test/unit/int_algolia/scripts/algolia/helper/orderHelper.test.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+// Mock the Site module
+jest.mock('dw/system/Site', () => {
+    return {
+        getCurrent: function () {
+            return {
+                getAllowedLocales: function () {
+                    var arr = ['default', 'fr', 'en'];
+                    arr.size = function () {
+                        return arr.length;
+                    };
+                    return arr;
+                }
+            };
+        }
+    }
+}, { virtual: true });
+
+// Mock AlgoliaLocalizedProduct
+jest.mock('*/cartridge/scripts/algolia/model/algoliaLocalizedProduct', () => {
+    return function(config) {
+        return {
+            objectID: config.product ? config.product.ID : 'test-product-id',
+            locale: config.locale,
+            in_stock: config.product ? config.product.in_stock : true,
+            price: {
+                USD: 129.99,
+                EUR: 119.99
+            },
+            _attributeList: config.attributeList
+        };
+    };
+}, { virtual: true });
+
+// Mock algoliaData
+jest.mock('*/cartridge/scripts/algolia/lib/algoliaData', () => {
+    return {
+        calculateIndexName: function (indexType, locale) {
+            return indexType + '_' + locale;
```

Comment:
Why do you need to mock this function? Doesn't it work with the current setup? All current tests are using the actual function without issue.

Reply:
removed